sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -302,6 +302,16 @@ enum class queue_state {
 };
 
 namespace property {
+
+namespace graph {
+
+class no_cycle_error_check {
```

Comment:
I think the word error in the property name is a bit superfluous and `no_cyle_check` would be a bit more concise.